site-src/implementations.md

```diff
@@ -30,6 +30,7 @@ cover, and documentation to help users get started.
 - [Kong][10] (GA)
 - [Kuma][11] (beta)
 - [LiteSpeed Ingress Controller][19]
+- [ngrok][33] (preview)
 - [NGINX Gateway Fabric][12] (GA)
```

Comment:
For neutrality we have historically done lexicographical ordering using the english alphabet, so this actually needs to go _after_ nginx:

```suggestion
- [NGINX Gateway Fabric][12] (GA)
- [ngrok][33] (preview)
```